tracer/src/Datadog.Trace/AppSec/Waf/NativeBindings/WafLibraryInvoker.cs

```diff
@@ -265,25 +266,25 @@ internal IntPtr ObjectMap()
             string message,
             ulong message_len)
         {
-            var formattedMessage = $"{level}: [{function}]{file}({line}): {message}";
+            var location = $"[{function}]{file}({line})";
```

Comment:
This is only constant for each invocation. At the expense of duplicating code, should we break this up and insert it into each log message instead?

Reply:
I combined these into a single value (which is passed as a parameter) because our logging APIs only go up to accepting 3 parameters 😅 We absolutely could add extra methods to ILogger just for this (there's a couple of other similar cases), but it didn't seem worth it? _Alternatively_, we could break it up and go the `new object[]{}` route. I'm not passionate about it either way (this seemed like a happy medium)